outputs.tf

```diff
@@ -0,0 +1,14 @@
+output "id" {
+  value       = "${aws_cloudtrail.trail.id}"
+  description = "The name of the trail."
```

Comment:
We don't seem to use `description` fields in other modules, but they're supported in HCL. For `variable` blocks, the `description` value is currently used on Terraform Registry to document input variables ([source](https://www.terraform.io/docs/configuration/variables.html#description)); looks like the `output` blocks are not yet used for anything but are planned for use in documentation in a future version of Terraform ([source](https://www.terraform.io/docs/configuration/outputs.html#description)).